### ISSUE TITLE
Model of Small Cooling Tower is rotated by 180 degree

### ISSUE THREAD

**Issue by cy-one** [MaFi-Games/Captain-of-Industry-issues#1653]
### Issue description

The model has a red pipe connection and a blue pipe connection. Intuitively, red means "hot steam in" and blue "water out", however it works the other way around. There's also arrows in the textures, that go from the red side to the blue side but again, the flow is the other way around.

### Reproduction steps

It's part of the model. It's 100% reproducable by... placing a tower :D

### Game version and build number

v0.6.4d b343

### Operating system

Windows

### Attachments

<img width="504" alt="Screenshot 2024-10-17 073052" src="https://github.com/user-attachments/assets/4c2747b3-5390-41fb-ad0e-7ad8c5d5495d">

**Comment by poltime** [comment]
In case you haven't noticed, everywhere the green arrow is the input, the red one is the output.

**Comment by JamieRCHI** [comment]
> In case you haven't noticed, everywhere the green arrow is the input, the red one is the output.

Yes. I don't believe the color of the arrows have anything to do with the contents that enter or leave a building.
